### PR TITLE
Include BCDevOps Backup-Container

### PR DESCRIPTION
**Pull request by DerekRoberts**
Auto-backups with backup-container.

---

Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-quickstart-typescript-594-backend.apps.silver.devops.gov.bc.ca/) available
[Frontend](https://nr-quickstart-typescript-594-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-quickstart-typescript/actions/workflows/merge-main.yml)

### CODE REVIEW

File: database/openshift.deploy.yml [hunk 1]
```diff
@@ -24,6 +24,60 @@ parameters:
...
+  - name: "FAILED_JOBS_HISTORY_LIMIT"
+    description: "The number of failed jobs that will be retained"
+    value: "2"
+    required: true
+  - name: "JOB_BACKOFF_LIMIT"
+    description: "The number of attempts to try for a successful job outcome"
+    value: "0"
+  - description: Volume space available for data, e.g. 512Mi, 2Gi.
+    name: PVC_SIZE
+    value: 1Gi
```
Comment (mishraomp) [inline database/openshift.deploy.yml:76]: let us skim it down a bit to 256Mi to help the team with only 1Gi as storage capacity in openshift realm.

File: database/templates/openshift.backups.yml [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  - name: "FAILED_JOBS_HISTORY_LIMIT"
+    description: "The number of failed jobs that will be retained"
+    value: "2"
+    required: true
+  - name: "JOB_BACKOFF_LIMIT"
+    description: "The number of attempts to try for a successful job outcome"
+    value: "0"
+  - description: Volume space available for data, e.g. 512Mi, 2Gi.
+    name: PVC_SIZE
+    value: 1Gi
```
Comment (mishraomp) [inline database/templates/openshift.backups.yml:60]: same for volume here.

**Review by mishraomp — APPROVED** [review]

File: database/templates/openshift.backups.yml (hunk 2 above)
Comment (DerekRoberts) [reply database/templates/openshift.backups.yml]: Thanks for the improvements!  Changes made.  :)

**Review by mishraomp — APPROVED** [review]
LGTM.

**Pull request merged**